pkg/console/subresource/configmap/configmap.go

```diff
@@ -59,7 +61,9 @@ func DefaultConfigMap(operatorConfig *operatorv1.Console, consoleConfig *configv
 	brand := operatorConfig.Spec.Customization.Brand
 	docURL := operatorConfig.Spec.Customization.DocumentationBaseURL
 	providers := operatorConfig.Spec.Providers
-	userDefinedConfig := NewYamlConfig(host, logoutRedirect, brand, docURL, apiServerURL, providers)
+	customProductName := operatorConfig.Spec.Customization.CustomProductName
+	customLogoFile := operatorConfig.Spec.Customization.CustomLogoFile.Name
```

Comment:
Don't you need to append the key to this path to get the full path?

Reply:
Yup:

```
 oc logs -f console-6694c875d5-rp8qw
2019/06/6 22:04:01 cmd/main: could not read logo file: stat my-logo-file: no such file or directory
```

Reply:
done